integration-cli/docker_api_volumes_test.go

```diff
@@ -87,4 +92,11 @@ func (s *DockerSuite) TestVolumesAPIInspect(c *check.C) {
 	c.Assert(status, checker.Equals, http.StatusOK, check.Commentf(string(b)))
 	c.Assert(json.Unmarshal(b, &vol), checker.IsNil)
 	c.Assert(vol.Name, checker.Equals, config.Name)
+
+	//comapring CreatedAt field time for the new volume to now. Removing a minute from both to avoid false positive
+	testCreatedAt, _ := time.Parse(time.RFC3339, strings.TrimSpace(vol.CreatedAt))
```

Comment:
I think we should check the error here, just in case.

```go
testCreatedAt, err := time.Parse(time.RFC3339, strings.TrimSpace(vol.CreatedAt))
c.Assert(err, checker.IsNil)
```